MagneticField/ParametrizedEngine/test/alpaka/testParabolicParametrizedMagneticField.dev.cc

```diff
@@ -0,0 +1,141 @@
+#include <alpaka/alpaka.hpp>
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/FWLite/interface/Record.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include "CLHEP/Geometry/Vector3D.h"
+
+#include "MagneticField/ParametrizedEngine/interface/alpaka/ParabolicParametrizedMagneticField.h"
+
+#include "FWCore/Utilities/interface/stringize.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+#include <iostream>
+#include <fstream>
+#include <Eigen/Core>
```

Comment:
Could you sort the includes, with the external first and the standard C++ ones at the top (though other prefer the opposite order, with the externals later and the standard C++ at the bottom) ?
Just... not randomly :)

Reply:
OK :)
https://github.com/cms-sw/cmssw/pull/44360/commits/c6936d662ee52343d6446d101f7660a5c8ddffda